### ISSUE TITLE
systemd and logging

### ISSUE THREAD

**Issue by pghmcfc** [proftpd/proftpd#1002]
This issue is referred from https://bugzilla.redhat.com/show_bug.cgi?id=1808989

The systemd unit file for starting proftpd in standalone mode includes:
```
Type = simple
ExecStart = /usr/sbin/proftpd --nodaemon $PROFTPD_OPTIONS
```
"Type = simple" means that systemd just execs the executeable and no forking takes place (hence the ExecStart line with the --nodaemon option). systemd then logs everything that comes out of the program.

As a result, we end up with log entries like these:
```
Mar 02 08:55:28 hostname proftpd[2839]: 2020-03-02 08:55:28,055 hostname proftpd[2839]: processing configuration directory '/etc/proftpd/conf.d'
Mar 02 08:55:28 hostname proftpd[2839]: 2020-03-02 08:55:28,064 hostname proftpd[2839] 93.185.100.237: ProFTPD 1.3.6b (maint) (built Tue Feb 4 2020 20:24:04 UTC) standalone mode STARTUP
```
Both systemd and proftpd add timestamps to the log message, which isn't really necessary. Is there a way (or could there be an option) to get proftpd not to add its own timestamp?

**Comment by Castaglia** [comment]
Well, is there a way to tell `systemd` not to add its own timestamp?  ProFTPD's timestamps include milliseconds, whereas it doesn't look like the `systemd`-provided ones do.

**Comment by pghmcfc** [comment]
Looks like the data in the journal does have the timestamps to microsecond precision so at least there wouldn't be loss of data, although the default logging would not include the milliseconds - you'd need the journalctl output for that, or you could use the default logformat rather than any systemd-specific one.

**Comment by Castaglia** [comment]
Just another example of why I hate logging configuration -- there's no consistency, and everyone wants _their_ preferred format to be used.  Whatever format is chosen will be compatible with someone else's setup.  See also [Bug #4185](http://bugs.proftpd.org/show_bug.cgi?id=4185).

**Comment by Castaglia** [comment]
[This PR](https://github.com/proftpd/proftpd/pull/1009) implements a LogOptions directive, which can be used to omit the timestamp, hostname, and <VirtualHost> fields from the ProFTPD syslog/module logging.

For example:
```
LogOptions -Timestamp -Hostname -VirtualHost
```
And similar, for trace logging:
```
TraceOptions -Timestamp
```

**Comment by Castaglia** [comment]
Considering this addressed, per PR #1009 for [Bug #4185](http://bugs.proftpd.org/show_bug.cgi?id=4185).
